### ISSUE TITLE
Alpha should apply to layer as a whole, not every element in layer

### ISSUE THREAD

**Issue by dominikh** [linebender/velato#37]
When modifying the with_winit example to render with a non-unity alpha, such as

```
renderer.render(lottie, frame, Affine::IDENTITY, 0.5)
```

the result looks like
![image](https://github.com/user-attachments/assets/8411297d-a026-4b36-ad3e-85e65e4959ad)
with each component of the tiger having alpha applied to it individually. Instead, the alpha should apply to the rendered image as a whole.

I suspect, but haven't confirmed, that the same issue exists for layers specified by the lottie file.

**Comment by simbleau** [comment]
This is known, and expected actually.

Here's a workaround for you:

```rust
let scene_buffer = Scene::new();
let scene = velato...render(...);


                            scene_buffer.push_layer(
                                vello::peniko::Mix::Normal,
                                *alpha,
                                *affine,
                                &vello::kurbo::Rect::new(
                                    0.0,
                                    0.0,
                                    asset.width as f64,
                                    asset.height as f64,
                                ),
                            );
                        }
                        scene_buffer.append(scene, None);
                        scene_buffer.pop_layer();

```

**Comment by simbleau** [comment]
@dfrg or @DJMcNab or @waywardmonkeys  - Do we have any opinions about whether we should change the render method to match user expectations for alpha?

**Comment by dominikh** [comment]
Considering this also applies to layers and groups with transparency, what would be the reason for _not_ changing the behavior? Just to reduce the number of clip layers? Because as is, Velato will not correctly render any layer or group with transparency that contains overlapping shapes. That is, this problem isn't specific to the `alpha` parameter of `append`.

**Comment by simbleau** [comment]
I think I agree, this could be easily fixed and a PR would be appreciated.

For historic context - If I recall correctly, some of the friction was that you need a shape to clip by, and that's not always correct or obvious. For a lottie file, we could use the `asset.width` and `asset.height` to clip by, but there's no bounding box you can query by for a general vello `Scene`. I think there was talk on making the shape optional, so we can guarantee correctness on lotties which spill outside of their widthxheight boxes.

**Comment by dfrg** [comment]
> @dfrg or @DJMcNab or @waywardmonkeys - Do we have any opinions about whether we should change the render method to match user expectations for alpha?

I’d say it should either be fixed or the parameter should be removed since the behavior is clearly incorrect. If lottie layers are exhibiting the same problem, which seems likely, we should surround them with vello layers when alpha is non-unity.

**Comment by DJMcNab** [comment]
I agree, we should probably use a Vello layer here. I think we probably should have an issue in Vello tracking the implementation of https://github.com/linebender/vello/issues/623
